src/test/java/org/folio/search/controller/SearchInstanceIT.java

```diff
@@ -61,7 +61,23 @@ void canSearchByInstances(String testName, String query, Object[] queryArguments
       arguments("search by hrid with wildcard (ends with)", "hrid=={value}", array("*00022"), null),
       arguments("search by hrid with wildcard (contains)", "hrid=={value}", array("*00000002*"), null),
       arguments("search by keyword that matches title", "keyword all {value}", array("primer"), null),
-      arguments("search by subjects", "subjects all {value}", array("semantic"), null)
+      arguments("search by subjects", "subjects all {value}", array("semantic"), null),
+      arguments("search by classification number",
+        "classifications.classificationNumber=={value}", array("025.04"), null),
+      arguments("search by classification number (using wildcard)",
+        "classifications.classificationNumber=={value}", array("025*"), null),
+      arguments("search by notes (word in note)", "notes.note all {value}", array("references"), null),
+      arguments("search by notes (partial match)", "notes.note all {value}", array("reference"), null),
+      arguments("search by notes (root of word)", "notes.note all {value}", array("readability"), null),
+      arguments("search by staffOnly", "notes.staffOnly=={value}", array("false"), null),
```

Comment:
I recall Magda asked if it is possible to implement search by public notes only (i.e. `staffOnly` is `false`), it is not possible, right?